content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -54,8 +54,8 @@ it can't be both.
 
 ClusterRoles have several uses. You can use a ClusterRole to:
 
-1. define permissions on namespaced resources and be granted within individual namespace(s)
-1. define permissions on namespaced resources and be granted across all namespaces
+1. define permissions on namespaced resources and be granted access within individual namespace(s)
+1. define permissions on namespaced resources and be granted access across all namespaces
```

Comment:
ClusterRoles are a set of permissions. ClusterRoles do not grant access. RoleBindngs and ClusterRoleBindings grant the permissions set by a ClusterRole to a user or set of users